### ISSUE TITLE
Add "Relay Attacks" to security considerations

### ISSUE THREAD

**Issue by OR13** [w3c/vc-data-model#1041]
covering cases where challenges are forwarded / related to holder binding / VPs, etc...

We should describe when relay is legitimate, and where it might cause problems.

relevant attacks on in person presentation, or man in the middle... etc.

**Comment by bumblefudge** [comment]
rePlay attacks or reLay attacks?

**Comment by OR13** [comment]
Related thread from RATs WG at IETF:

- https://mailarchive.ietf.org/arch/msg/rats/K4Sqb3aub6uJh0So55NueWsqhM4/

**Comment by awoie** [comment]
This issue is blocked by the holder binding discussion. It really depends on whether such a feature is available. It makes a difference for the security consideration section whether a VC has some protection against impersonation or not.

**Comment by OR13** [comment]
Closing this issue, I think it is not addressable in the core data model, it applies to securing formats.

**Comment by TallTed** [comment]
> Closing this issue, I think it is not addressable in the core data model, it applies to securing formats.

You opened this as a Security Consideration, which it remains, and which does not require addressing in the core data model — Security Considerations are about awareness and consideration regarding whatever deployment... 

So I don't understand its closure, as opposed to adding a few sentences to the Security Considerations section.

**Comment by TallTed** [comment]
> Closing this issue, I think it is not addressable in the core data model, it applies to securing formats.

You opened this as a Security Consideration, which it remains, and which does not require addressing in the core data model — Security Considerations are about awareness and consideration regarding whatever deployment... 

So I don't understand its closure, as opposed to adding a few sentences to the Security Considerations section.

**Comment by OR13** [comment]
@TallTed you are welcome to raise a new issue or file a PR that implements changes you want to see.
